### ISSUE TITLE
CSP Bypass (Low) can't be solved with pastebin anymore

### ISSUE THREAD

**Issue by kevindoubleu** [digininja/DVWA#382]
Pastebin includes this header in it's responses
```
x-content-type-options: nosniff
```
which causes this error in the developer console
![image](https://user-images.githubusercontent.com/54675998/92683601-80678f00-f35d-11ea-9527-7294d74196cb.png)
the pastebin link: [here](https://pastebin.com/raw/wYDrwL2d)
Now the script won't execute because that header will look at this other header and determine that it's plaintext and not js code to be executed
```
Content-Type: text/plain; charset=utf-8
```
After fiddling with this problem for some days, looking at writeups, etc. I'm guessing solving this isn't possible as we can't remove this header from pastebin ?

**Comment by digininja** [comment]
You are correct, they must have updated something to add the header. I've just had a quick look at some of the alternatives and none were appropriate. I'll put a note on the challenge and keep looking. If you can find something that will let you post content and that doesn't set the header, let me know.

**Comment by digininja** [comment]
Update done in https://github.com/digininja/DVWA/commit/9da93a0f49a774e28d58dc35e4708787e5851c95

**Comment by kevindoubleu** [comment]
Thank you for your response, I've roamed around a bit and found hastebin, tried it out and looks like it works fine.

**Comment by kevindoubleu** [comment]
Resolved in b445942
